File: Lugar.java
public class Lugar {
    private float longitud;
    private float latitud;
    private float radio;

    public Lugar(float longitud, float latitud) {
        this.longitud = longitud;
        this.latitud = latitud;
        this.radio=100;
    }

    public float getLongitud() {
        return longitud;
    }

    public float getLatitud() {
        return latitud;
    }

    public float getRadio() {
        return radio;
    }
}

File: app_objetos_perdidos/lib/utils/lugar.dart
import 'package:hive/hive.dart';
part 'lugar.g.dart';

@HiveType(typeId: 1)
class Lugar {
  @HiveField(0)
  double latitud;
  
  @HiveField(1)
  double longitud;
  
  @HiveField(2)
  int radio;

  Lugar(this.latitud, this.longitud, this.radio);
}